aiida_common_workflows/workflows/relax/vasp/generator.py

```diff
@@ -29,12 +43,12 @@ def __init__(self, *args, **kwargs):
 
     def _initialize_protocols(self):
         """Initialize the protocols class attribute by parsing them from the protocols configuration file."""
-        with open(str(pathlib.Path(__file__).parent / 'protocol.yml')) as handle:
+        with open(str(pathlib.Path(__file__).parent / 'protocol.yml'), encoding='UTF-8') as handle:
```

Comment:
You can also use `importlib.resources`. After putting

```
from importlib import resources
```

at the top of the module, you can use:

```suggestion        
        from .. import vasp
        with resources.open_text(vasp, 'protocol.yml') as handle:
```